news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+What would you call the greatest gift that osu! has given you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+osu! has given me so many opportunities in my life that it's crazy when I look back on it. Not only do I have the opportunity to travel and meet people around the world, have fun and make a career out of something I love AND feel like I have the drive to better my content and my skills, osu! also gave me the greatest gift of being able to connect with people. 
+
+I'm not very social and I keep to myself most of the time, but I've made long lasting friendships during my entire time on osu!. I feel like I've gained a family. The community can seem toxic at times online, but from my experience, it's never the reality when you get to know people. Not only that, but building my own little community has also allowed people I know to make great connections and friendships with other people. When I take a step back, I feel like a little spider that's woven a big web of connections to all different kinds of aspects of my life. Without osu! I would be very different!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Did streaming push you into becoming a top player, or was it vice versa?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+I think it kind of goes both ways, but I totally started streaming and doing content creation because of the drive to become a top player. I have practically never had an osu! session off stream since I got unbanned in 2018 (that's a whole other story). I started posting on YouTube shortly after my first OWC appearance in late 2020, which really kickstarted my career. I wanted to do something every week that would get people interested. Be that FCing every map in my first match, or carrying my team to victory by taking extra mods on FreeMod picks. That ended up giving me a lot of traction on Twitch, and I used that as leverage to keep pushing my competitive skill and get better!
```

Comment:
note that we usually never mention the topic of restrictions in news posts

Reply:
I understand that, but let's be honest, it really is an important part of many players' history

Reply:
personally id leave it but lets see if ppy has anything against it

Reply:
> it's fine, although leaving out the reason is weird. did they cheat?

She was falsely banned like many players in the past, like WhiteCat if that changes things